UniWatch/Controllers/ClassController.cs

```diff
@@ -108,11 +107,11 @@ public ActionResult Delete(int classId)
         public ActionResult DeleteConfirmed(int classId)
```

Comment:
Authorization attribute is controller wide. Only a teacher can get here. Was tested.